frontends/precinct-scanner/src/api/config.test.ts

```diff
@@ -32,6 +32,14 @@ test('DELETE /config/election to delete election', async () => {
   await config.setElection(undefined);
 });
 
+test('DELETE /config/election ?ignoreBackupRequirement query param', async () => {
+  fetchMock.deleteOnce(
+    '/config/election?ignoreBackupRequirement=true',
+    JSON.stringify({ status: 'ok' })
+  );
+  await config.setElection(undefined, { ignoreBackupRequirement: true });
```

Comment:
what are we testing here? Is there any way for this test to fail? Maybe just missing an assert?

Reply:
Good q! I was following the pattern of the test above this one:

https://github.com/votingworks/vxsuite/blob/a3f1c3378d83adcd70a2113558f48490f4fff18b/frontends/precinct-scanner/src/api/config.test.ts#L30-L33

The test I've added _will_ fail if an unmatched call is made, i.e. a call to a URL other than `/config/election?ignoreBackupRequirement=true`. That said, looks like the test _won't_ fail if no call at all is made - we could definitely add an assert to make sure a call is made.

We should in fact make that change in many places in the codebase. I can open a tech improvement ticket to fill in this gap across vxsuite.